hera_mc/args_test.py

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2016 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""This is meant to hold utility scripts for handling parts and connections
+
+"""
```

Comment:
It is not clear to me what this module is for either. Please elaborate in the docstring.